microsoft_atp/plugin.spec.yaml

```diff
@@ -21,481 +21,441 @@ hub_tags:
+     creationDateTimeUtc:
+       title: "Creation Date Time UTC"
+       type: string
+       description: "Creation date time UTC"
+       required: false
+     errorHResult:
+       title: "Error HResult"
+       type: integer
+       description: "Error HResult"
+       required: false
+     id:
+       title: "ID"
+       type: string
+       description: "ID"
+       required: false
+     lastUpdateDateTimeUtc:
+       title: "Last Update Date Time UTC"
+       type: string
+       description: "Last update date time UTC"
+       required: false
+     machineId:
+       title: "Machine ID"
+       type: string
+       description: "Machine ID"
+       required: false
+     requestor:
+       title: "Requestor"
+       type: string
+       description: "Requestor"
+       required: false
+     requestorComment:
+       title: "Requestor Comment"
+       type: string
+       description: "Requestor comment"
+       required: false
+     status:
+       title: "Status"
+       type: string
+       description: "Status"
+       required: false
+
 connection:
-  auth_url:
-    title: Authorization server URL
-    description: 'Server used to obtain OAuth token. Example: https://login.windows.net/{Azure
-      Tenant ID}/oauth2/token'
-    required: true
+  application_id:
+    title: Application ID
+    description: Application (client) ID
     type: string
-  client_id:
-    title: Client ID
-    description: Client ID obtained from Windows Defender Security Center API settings
     required: true
-    type: string
-  client_secret:
-    title: Client Secret
-    description: Client secret obtained when authorizing an application to connect
-      to Windows Defender
+    example: xxxxxxx-xxxxxx-xxxxx-xxxx
+    order: 1
+  application_secret:
+    title: Application Secret
+    description: Application secret
+    type: credential_secret_key
     required: true
+    example: xxxxxxxxxxxxxxxxxxxxx
+    order: 2
+  directory_id:
+    title: Directory ID
+    description: Directory (tenant) ID
     type: string
-  host:
-    title: Host URL
-    description: Base URL for endpoints, e.g. https://wdatp-alertexporter-us.securitycenter.windows.com
     required: true
-    type: string
-    default: https://wdatp-alertexporter-us.securitycenter.windows.com
-  resource_url:
-    title: Resource URL
-    description: This will be the server that will return information from the plug-in
-      requests. This will typically be https://graph.windows.net (for alert information)
-      or https://api.securitycenter.windows.com (for machine information and actions).
-    required: false
-    type: string
-    default: https://api.securitycenter.windows.com
-  seconds_ago:
-    title: Seconds Ago
-    description: Seconds before the first run of the trigger to check for alerts.
-      If left blank, the trigger will start scanning from the time the workflow begins.
-    required: false
-    type: integer
-    default: 0
+    example: xxxx-xxxx-xxxx-xxxx-xxx
```

Comment:
Do we have a better example that shows the char set?

Reply:
I pulled the values from the MS Intune app.